libs/constants/index.js

```diff
@@ -87,7 +87,7 @@ const scriptSrcValues = [
   // - Previous hash (to avoid cache invalidation issues):
   "'sha256-GA8+DpFnqAM/vwERTpb5zyLUaN5KnOhctfTsqWfhaUA='",
```

Comment:
We need to copy the previous "Current hash" here:

```suggestion
  "'sha256-ryRT1qGpDn+Y+tJ6gmb6XWaV28URicWtYzcBXNv1i+U='",
```